app/interfaces/api/logger.rb

```diff
@@ -19,9 +19,9 @@ def log_api_response(status)
   end
 
   def response_body
-    JSON.parse(@app_response.body.first)
+    JSON.parse(@app_response[2].first)
```

Comment:
so [Rack::Response is defined here](https://github.com/rack/rack/blob/master/lib/rack/response.rb#L42) and [grape calls it here](https://github.com/ruby-grape/grape/blob/master/lib/grape/middleware/base.rb#L61) 

mightbe worth trying if 
```
@app_respinse.is_a?(Rack::Response)
```
as you could then use that classes interface